### DISCUSSION TITLE
ordinal_features

### DISCUSSION THREAD

**Discussion by lalitsc12** [pycaret/pycaret#3707] [General]
Pycaret treats some features as ordinal_features.  how to override the default behaviour.

**Comment by celestinoxp** [comment]
@ngupta23 Do you want to respond?

**Comment by moezali1** [comment]
@lalitsc12 You have to use `ordinal_features` parameter in the `setup`.

Read more: https://pycaret.gitbook.io/docs/get-started/preprocessing/data-preparation#ordinal-encoding

**Reply by lalitsc12** [reply]
Hello. I have set ordinal feature  to None still pycaret takes some features as ordinal instead of categorical

**Reply by tonpimenta** [reply]
Were you able to solve this? Im running on the same problem. I don't have ordinal features and PyCaret keeps taking some as ordinals. I'm wondering if this is a problem with the fact I'm running on Colab

**Comment by lalitsc12** [comment]
> @lalitsc12 You have to use `ordinal_features` parameter in the `setup`.
> 
> Read more: https://pycaret.gitbook.io/docs/get-started/preprocessing/data-preparation#ordinal-encoding

**Comment by tonpimenta** [comment]
I believe this question is related to this closed issue [3846](https://github.com/pycaret/pycaret/issues/3846) #3846
